### PR TITLE
Begin acceptance test for scheduling a one-time payment

### PR DESCRIPTION
**Pull request by lukemelia**
While there is still more work to do before we can schedule a payment via the UI, there are enough changes in this PR that I think it is a good time for a review and merge.

Here's the mock and the current status:

Mock:

Payment Total - Information entered

As of this PR:

image

### CODE REVIEW

File: packages/safe-tools-client/app/components/schedule-payment-form-action-card/validator.ts [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+  maxGasFee: 'normal' | 'high' | 'max' | undefined;
+}
+
+export default class SchedulePaymentFormValidator {
+  form: ValidatableForm;
+
+  @tracked isRecipientAddressInvalid = false;
+  @tracked recipientAddressErrorMessage = '';
+  @tracked isPaymentAmountInvalid = false;
+  @tracked paymentTokenErrorMessage = '';
```
Comment (backspace) [inline packages/safe-tools-client/app/components/schedule-payment-form-action-card/validator.ts:31]: Are these used anywhere? Maybe this is a copypaste relic 🤔

**Review by backspace — APPROVED** [review]
happy to see this 🎉

File: packages/safe-tools-client/app/components/schedule-payment-form-action-card/validator.ts (hunk 1 above)
Comment (lukemelia) [reply packages/safe-tools-client/app/components/schedule-payment-form-action-card/validator.ts:31]: It is more aspirational :). Open question at the moment: besides disabling the button, what other "invalid" state do we want to show and when?

Comment (backspace) [reply packages/safe-tools-client/app/components/schedule-payment-form-action-card/validator.ts:31]: oh yes, having an invalid state without indicating what needs to change can be so frustrating!

**Pull request merged**